project/api/write.py

```diff
@@ -11,7 +11,7 @@
 from api.forms import UpdateProfileImage
 from django.core.files import File   # need this for image file handling
 from django.core.files.base import ContentFile
-from api.models import Thread
+from api.models.thread import Thread
```

Comment:
As far as I can tell, `.models.thread` should work here, since they are in the same module namespace.